.github/workflows/run-danger.yml

```
@@ -1,13 +1,18 @@
-name: ☢️ Danger
+name: ☢️ Trigger Danger On Buildkite
 
 on:
   pull_request:
-    types: [opened, reopened, ready_for_review, synchronize, edited, review_requested, review_request_removed, labeled, unlabeled, milestoned, demilestoned]
+    types: [labeled, unlabeled, milestoned, demilestoned]
```

I noticed we have [this check](https://github.com/wordpress-mobile/release-toolkit/blob/ffacf8a24ce7875012ed8d1cacccb1cda2ddf17b/Dangerfile#L52) in the `Dangerfile`, which should update according to the "review requested" events.

```suggestion
    types: [labeled, unlabeled, milestoned, demilestoned, review_requested, review_request_removed]
```

**Alternatively**, we could set a team to be auto-assigned for PRs, like we have in some other repos like PocketCasts. I think it probably makes sense for this repo?

---

Maybe we can do both?

---

Not fully sure if both makes sense as the automatically assigned team would always be assigned... then in this case maybe the check would be relevant only if someone unassigns the team? 🤔